apps/epic-react/src/templates/lesson-template.tsx

```diff
@@ -90,6 +90,8 @@ const ExerciseTemplate: React.FC<{
       lesson._type === 'interview') &&
     session
 
+  const epicReactModule = {...module, moduleType: 'module'}
```

Comment:
```suggestion
// epic react modules are always type 'module'
  const epicReactModule = {...module, moduleType: 'module'}
```

because the lesson list is data driven, this is the only way to change the url without either changing the source code in `Collection` that uses `lessonPathBuilder` to build the dynamic path. 

This seems cleaner to me than redefining the `lessonPathBuilder`